receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -116,6 +144,69 @@ metrics:
       value_type: int
     attributes: []
     enabled: true
+  elasticsearch.node.fs.disk.total:
+    description: The total amount of disk space across all file stores for this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.fs.disk.free:
+    description: The total number of unallocated disk space across all file stores for this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.fs.disk.operations:
+    description: The total number of read and write operations across all file stores for this node.
+    unit: "{operations}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [  ]
+    enabled: true
+  elasticsearch.node.fs.disk.read_operations:
+    description: The total number of read operations across all file stores for this node.
+    unit:  "{operations}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.fs.disk.write_operations:
+    description: The total number of write operations across all file stores for this node.
+    unit:  "{operations}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.fs.disk.read_bytes:
+    description: The total number of kilobytes read across all file stores for this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.fs.disk.write_bytes:
```

Comment:
Is `fs.disk` very meaningful here? Maybe just `disk`?

Reply:
The metric naming I've been using here and throughout my PR uses the same key names from the `_nodes/_all/stats` JSON response body. I find using the same names is easier to track what metrics are being used instead of looking at the code to determine the metric-to-JSON mapping.

Is there a preference to be concise and less wordy?

Reply:
Thanks for the explanation. I'll stick with being concise in the metric naming.